zebra-crosslink/src/mal_system.rs

```diff
@@ -777,7 +777,7 @@ impl FatPointerToBftBlock {
             return None;
         }
         let vote_for_block_without_finalizer_public_key = bytes[0..76 - 32].try_into().unwrap();
-        let len = u16::from_le_bytes(bytes[76 - 32..2].try_into().unwrap()) as usize;
+        let len = u16::from_le_bytes(bytes[2..76 - 32].try_into().unwrap()) as usize;
```

Comment:
What is the purpose of this change?

Reply:
It came from a clippy check

Reply:
Looking at it now I think I just did it reflexively in the midst of the rebasing, I could just ignore the check like I did the `never_loop` check